docs/install.md

```diff
@@ -17,51 +17,63 @@ The Linux kernel (and optionally initramfs) is embedded in the container image;
 in that directory.
 
-The `bootc install` command bridges the two worlds of a standard runnable OCI image
-and a bootable system by running tooling
-logic embedded in the container image to create the filesystem and
-bootloader setup dynamically, using tools already embedded in the container
-image.  This requires running the container via `--privileged`; it uses
-the running Linux kernel to write the file content from the running container image;
-not the kernel inside the container.
+The `bootc install` and `boot install-to-filesystem` commands bridge the two worlds
+of a standard, runnable OCI image and a bootable system by running tooling logic embedded
+in the container image to create the filesystem and bootloader setup dynamically.
+This requires running the container via `--privileged`; it uses the running Linux kernel
+on the host to write the file content from the running container image; not the kernel
+inside the container.
 
-However nothing *else* (external) is required to perform a basic installation
-to disk.  This is motivated by experience gained from the Fedora CoreOS
+However, nothing *else* (external) is required to perform a basic installation
+to disk.  (The one exception to host requirements today is that the host must
+have `skopeo` installed.  This is a bug; more information in
+[this issue](https://github.com/containers/bootc/issues/81).)
+
+This is motivated by experience gained from the Fedora CoreOS
 project where today the expectation is that one boots from a pre-existing disk
-image (AMI, qcow2, etc) or use [coreos-installer](https://github.com/coreos/coreos-installer)
+image (AMI, qcow2, etc) or uses [coreos-installer](https://github.com/coreos/coreos-installer)
 for many bare metal setups.  But the problem is that coreos-installer
 is oriented to installing raw disk images.  This means that if
 one creates a custom derived container, then it's required for
 one to also generate a raw disk image to install.  This is a large
 ergonomic hit.
 
-With `bootc install`, no extra steps are required.  Every container
+With the `bootc` install methods, no extra steps are required.  Every container
 image comes with a basic installer.
 
 ## Executing `bootc install`
 
-The installation command must be run from the container image
+The two installation commands allow you to install the container image
+either directly to a block device (`bootc install`) or to an existing
+filesystem (`bootc install-to-filesystem`).
+
+The installation commands **MUST** be run **from** the container image
 that will be installed, using `--privileged` and a few
-other options.
+other options. This means you are (currently) not able to install `bootc`
+to an existing system and install your container image. Failure to run
+`bootc` from a container image will result in an error.
 
-Here's an example:
+Here's an example of using `bootc install` (root/elevated permission required):
```

Comment:
It would be great to have copy-paste examples in the docs.  I have encountered a number of issues trying to bootc-install in a pet VM.
"Boot this image, then do ..., then do a `podman-run ...bootc install ...`.